app/src/main/java/com/hyperether/getgoing/repository/room/entity/DbNode.java

```diff
@@ -29,6 +29,8 @@
     private float velocity;
     @ColumnInfo(name = "number")
     private long index; // node index within a particular route
+    @ColumnInfo
```

Comment:
@zukovicfilip why is this field added? There is "index" field that shows node index inside route, and we could maybe add "timestamp" field for each node. We can also change app behavior a little bit, so everything is saved in DB, and "save" route option becomes redundant. In that case user can start route, finish route and delete route. But everything is already saved in DB, as it is right now.

Reply:
When user pause tracking, one more node is added to the route and it is marked as last in that field. Later that information is used when drawing route on map to indicate algorithm to not draw line between that node and first node in next sequence.